python/lsst/pipe/base/executionButlerBuilder.py

```diff
@@ -339,7 +340,7 @@ def _setupNewButler(
 def _import(
     yamlBuffer: io.StringIO,
     newButler: Butler,
-    inserts: DataSetTypeMap,
+    inserts: DataSetTypeRefMap,
     run: Optional[str],
```

Comment:
We no longer use this parameter and `buildExecutionButler` also needs to be documented to ignore the `run` parameter. We now require that it's all matching the graph.